decomposed/extensions/MDDE_LDM/Profile/ExtendedSubObject/Stereotypes/mdde_SourceObject/Extended_Attributes/mdde_NumberOfJoinConditions.xml

```diff
@@ -9,20 +9,16 @@
 <a:ComputedAccessType>RO</a:ComputedAccessType>
 <a:GetMethodScript>Function %Get%(obj)
    Dim joinConditions
```

Comment:
Dim colJoinConditions : Set colJoinConditions  = obj.GetExtendedCollection(&quot;mdde_JoinConditions&quot;, False)

If colJoinConditions  Is Not Nothing Then
%Get% = colJoinConditions.Count
Else
%Get% = 0
EndIf